src/Core/src/Handlers/MenuFlyoutSubItem/MenuFlyoutSubItemHandler.iOS.cs

```diff
@@ -31,6 +31,18 @@ protected override UIMenu CreatePlatformElement()
 			return menu;
 		}
 
+		public static void MapIsEnabled(IMenuFlyoutSubItemHandler handler, IMenuFlyoutSubItem view)
+		{
+			if (handler.PlatformView is null)
+				return;
+
+			foreach (var menuElement in handler.PlatformView.Children)
```

Comment:
This is looping over all the items, so what happens when there are more levels? Also, is the IsEnabled not propagating to all the children in the xplat code? Like setting the menu item disabled should update all the properties on all the children in the tree.

Reply:
Also, what happens with Command.CanExecute?

Reply:
Really good points. For that the PR is still a Draft, keep working on more samples, covering cases, edge cases, etc.